CMakeLists.txt

```diff
@@ -66,7 +65,6 @@ if (LEGACY_BUILD)
     option(BUILD_OPTEL_OTLP_BENCHMARKS "Enables building the benchmark tests with open telemetry OTLP clients" OFF)
     option(USE_TLS_V1_2 "Set http client to enforce TLS 1.2" ON)
     option(USE_TLS_V1_3 "Set http client to enforce TLS 1.3" OFF)
-    option(AWS_USE_CRYPTO_SHARED_LIBS "Forces FindCrypto to use a shared crypto library if found. regardless of the value of BUILD_SHARED_LIBS" OFF)
```

Comment:
both [CRT](https://github.com/awslabs/aws-c-cal/blob/main/CMakeLists.txt#L16) and [S2N](https://github.com/aws/s2n-tls/blob/main/CMakeLists.txt) have a equivalent but "different" arg, since they are the same value, we can safely remove this now and let the downstream variables be the location for further changes.